### PR TITLE
table of results similar to fairtally

### PR DESCRIPTION
**Pull request by jspaaks**
## Description

Refs: #36

There is still a problem with permissions on the .tortellini directory (where I chose to write index.html). I'm currently bypassing that issue by manually setting the permissions on that directory so the file can be written.

If you run it on `urls.txt` with:

```text
https://github.com/nlesc-nano/insilico-server
https://github.com/nlesc-nano/Moka
https://github.com/nlesc-nano/flamingo
https://github.com/nlesc-nano/CAT
https://github.com/fair-workflows/nanopub
https://github.com/puregome/notebooks
https://github.com/matchms/ms2deepscore
https://github.com/GooglingTheCancerGenome/sv-channels/
https://github.com/fair-software/fairtally
https://github.com/e-mental-health/orange-hackathon
https://github.com/e-mental-health/data-processing
https://github.com/LINNAE-project/SFB-Annotator
https://github.com/DeepRank/deeprank
https://github.com/NLeSC/eEcology-Classification
https://github.com/parallelwindfarms/byteparsing
https://github.com/RETURN-project/BenchmarkRecovery
https://github.com/nlesc-nano/ceiba
https://github.com/NLeSC-GO-common-infrastructure/marzipan
https://github.com/FAIR-data-for-CAPACITY/FHIR-to-CAPACITY
https://github.com/JungleComputing/rocket
```

You should get this result in `.tortellini/index.html`:

image

* * *

## Before you get started

- [x] I read and I followed [contributing guidelines](https://github.com/tortellini-tools/action/blob/main/CONTRIBUTING.md)
- [x] [☝ Create an issue](https://github.com/NLeSC/licenseguard/issues/new/choose) to discuss what you are going to do

## Checklist

This section should be filled in by the creator of this pull request to make sure the pull request is ready to review.

- [x] This pull request has a descriptive title
- [x] Code is written according to the code quality guidelines
- [x] Documentation if applicable is available
- [x] Tests run successfully
- [x] All checks below this pull request were successful

## Instructions to review the pull request

1. &nbsp;

    ```
    mkdir .tortellini # to circumvent permission problem
    ```
1. Make `urls.txt` with above content.
1. Add environment vars to your terminal using my notes from https://github.com/tortellini-tools/action/issues/124#issue-914934346 (probably needs `export` too), then

    ```
    npm install
    npm run build
    npm run package
    node dist/index.js
    ```

### CODE REVIEW

File: src/config.ts [hunk 1]
```diff
@@ -1,8 +1,8 @@
 import * as core from '@actions/core'
 import * as io from '@actions/io'
 import * as fs from 'fs'
-import path from 'path'
 import fetch from 'node-fetch'
+import * as path from 'path'
```
Comment (fdiblen) [inline src/config.ts]:
```suggestion
import path from 'path'
```

Comment (jspaaks) [reply src/config.ts]: done in commit

**Review by fdiblen — APPROVED** [review]
Well done @jspaaks ! 
I could generate a report and everything looks good.
🥅

**Comment by jspaaks** [comment]
Thanks for the review! :soccer:

**Pull request merged**